frontend/src/pages/AboutUs.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+
+//style
+import "../css/pages/AboutUs.scss";
+
+
+//Images
+import linkedIn from "../images/linkedin 1.png";
+import img_1 from "../images/img-1.png";
+import img_2 from "../images/img-2.png";
+class Admin extends Component {
+
+    render() {
+        return (
+            <>
+                <div style={{ background: "#FFFDF6" }}>
+                    <div className="about">
+                        <div className="grid-container grid-x grid-margin-x">
+
+                            <div class="cell medium-4">
+                                <h1>About us</h1>
+                            </div>
+                            <div class="cell medium-8">
+                                <p>Maybelyn Plecic and Rashmeet Kaur are the founders of the Feminist Bible.
+                                It began in January
+                                2020, where they decided to co-write a book to share real women's stories.  In time, they
+                                realized that there were not enough repositories online that could easily share famous women
+                                role models from various fields. </p>
+
+                                <p>For many years, they both learned that most stories are from
+                                the male population. They both agreed it was time for the other half of the missing women's
+                                stories to share successes, struggles, inspirations, and journeys.
+                               </p>
+                            </div>
+                        </div>
+                    </div>
+
+
+                </div>
+
+                <div style={{ background: "linear-gradient(180deg, #FFFDF6 0%, #FFEBF3 100%)" }}>
+                    <div className="coFounder">
+                        <div className="grid-container grid-x grid-margin-x ">
+                            <div className="cell ">
+                                <h1 >
+                                    Co-Founders
+                            </h1>
+                                <br />
+                            </div>
+
+                            <div className="cell medium-6" style={{ paddingRight: "60px" }}>
```

Comment:
never add custom styles to foundation classes. Remove paddingRight. If the code breaks, it's fine. I'll fix it.

Reply:
I have changed it kindly review it